docs/building-assets.md

```diff
@@ -0,0 +1,98 @@
+# Building assets
+
+## Overview
+
+Toolkit provides a way to build theme assets with Gulp.js.
+
+By default a gulpfile is inclued and as well some npm packages in order to:
+
+- Look for Scss files and convert them into Css
+- Minify Css and Js
+- Merge files into one minimized file
+
+## How to use
+
+### Source files
+
+The folder structure for the source files should be aligned like this:
+
+- /theme_folder/css
```

Comment:
the folders are not correct, missing scss, we can like
```
- {your-theme-folder}/src/scss
- {your-theme-folder}/src/js
```

JS folder is commonly used by developer as folder to deliver js